forbidigo/forbidigo.go

```diff
@@ -36,13 +37,17 @@ func (a UsedIssue) Position() token.Position {
 
 func (a UsedIssue) String() string { return toString(a) }
 
-func toString(i Issue) string {
-	return fmt.Sprintf("%s at %s", i.Details(), i.Position())
+func toString(i UsedIssue) string {
+	s := fmt.Sprintf("%s at %s", i.Details(), i.Position())
+	if i.customMsg != "" {
+		s += ": " + i.customMsg
```

Comment:
The custom message goes on the end of the stringified issue. That seems fine to me, although I'm open to suggestion!